src/main/java/org/jahia/modules/dxwebpack/Main.java

```diff
@@ -106,6 +105,7 @@ public String getContextPath() {
             List<String> scripts = getApplicationScripts(appName);
             wrapper.setAttribute("scripts", "[" + StringUtils.join(scripts, ",") + "]");
 
+            response.setHeader("Cache-Control", "no-cache");
```

Comment:
no java test ?

Reply:
no

Reply:
in this commit I fixed the same issue by 
https://github.com/Jahia/jahia-ui-root/commit/de4e647e12f169f775690207c9200496bac7dc12
Using no-store instead of no-cache to avoid storing anything on the client side
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
Sorry i missed it ..  no-store or no-cache, looks good also

Reply:
```suggestion
            response.setHeader("Cache-Control", "no-store");
```

Reply:
like that ?